third-culture-software/bhima
# Add option to asset scan registry to show only the latest scan for each asset

This PR adds a custom search option to the asset scan registry to limit the display to only the latest (last) scan for each asset.

Closes https://github.com/IMA-WorldHealth/bhima/issues/6536

**TESTING**
- Use bhima_test
- Go to:  Asset Management > Asset Scan Registry
- Notice that there is one scan for each of the two items
- Add more scans to those two items (or others as desired).  Be sure to add notes in the scan modal to help you know which one is the latest.
- The registry should show all scans
- In the Search modal select the checkbox option at the bottom "Show only last scan"
- Now notice that only the lastest/last scan of each asset is shown
- Notice the filter item for this.  When you delete it, the display should revert to the normal display
